lib/cylc/cfgspec/suite.py

```diff
@@ -66,6 +67,10 @@ def _coerce_cycleinterval( value, keys, args ):
 
 def _coerce_cycletime( value, keys, args ):
     """Coerce value to a cycle point."""
+    if value == "now":
+        value = datetime.datetime.now().strftime("%Y%m%dT%H%M")
```

Comment:
Yeah, that's why I documented to use UTCnow if the suite is in UTC mode but the host isn't.  But getting the time zone from ```cylc.wallclock``` is a better idea.